src/webapp/src/client/app/wall/wall.controller.js

```diff
@@ -3,24 +3,38 @@
 
     angular
         .module('simoonaApp.Wall')
+        .constant('minDisplaySizes', {
+            rightSidebar: 992
```

Comment:
Looks like magic number.. Why 992 and not 994 or 892?
I understand, that value might be coming from some CSS, but in this case concerns would be mixed.
How do you imagine you would maintain such code if CSS/styles would be liked across whole Javascript code?..

Reply:
Yes, I understand this, I just haven't solved this yet:/